config/servicemappings/alloydb.yaml

```diff
@@ -67,6 +67,8 @@ spec:
         labels: labels
       resourceID:
         targetField: cluster_id
+      mutableButUnreadableFields:
+        - initial_user
```

Comment:
Is this field supposed to be mutable but unreadable? 

i.e. It is not a bug in the GCP API and we don't have plan to return it in the API response in future?

Reply:
We do not plan to return it in response, as it contains password as well. Also the API documentation clearly mentions that this is an input_only field